client/cli/src/lib.rs

```diff
@@ -229,78 +227,81 @@ pub trait SubstrateCli: Sized {
 }
 
 /// Initialize the logger
-pub fn init_logger(pattern: &str) {
-	use ansi_term::Colour;
-
-	let mut builder = env_logger::Builder::new();
-	// Disable info logging by default for some modules:
-	builder.filter(Some("ws"), log::LevelFilter::Off);
-	builder.filter(Some("yamux"), log::LevelFilter::Off);
-	builder.filter(Some("cranelift_codegen"), log::LevelFilter::Off);
-	builder.filter(Some("hyper"), log::LevelFilter::Warn);
-	builder.filter(Some("cranelift_wasm"), log::LevelFilter::Warn);
-	// Always log the special target `sc_tracing`, overrides global level
-	builder.filter(Some("sc_tracing"), log::LevelFilter::Trace);
-	// Enable info for others.
-	builder.filter(None, log::LevelFilter::Info);
+pub fn init_logger(
+	pattern: &str,
+	tracing_receiver: sc_tracing::TracingReceiver,
+	tracing_targets: Option<String>
+) {
+	match tracing_log::LogTracer::init() {
+		Ok(_) => (),
+		Err(_) => log::info!(
+			"💬 Not registering Substrate logger, as there is already a global logger registered!"
+		),
+	}
```

Comment:
```suggestion
	if tracing_log::LogTracer::init().is_err() {
		log::debug!(
			"💬 Not registering Substrate logger, as there is already a global logger registered!"
		);
	}
```

We should not print this as a `info` log line.